deploy/e2e.jsonnet

```diff
@@ -12,11 +12,24 @@ function(version='v0.0.1-alpha.3')
     },
   };
 
+  local server = (import 'github.com/parca-dev/parca/deploy/lib/parca/parca.libsonnet')({
+    name: 'parca',
+    namespace: ns.metadata.name,
+    image: 'ghcr.io/parca-dev/parca:' + self.version,
+    version: serverVersion,
+    replicas: 1,
+    corsAllowedOrigins: '*',
+  }) {
+    // Hide PSP: Removed in K8s 1.25
+    // TODO: Remove from lib or add option to to toggle it
+    podSecurityPolicy:: {},
+  };
+
   local agent = (import 'parca-agent/parca-agent.libsonnet')({
     name: 'parca-agent',
     namespace: ns.metadata.name,
     version: version,
-    image: 'ghcr.io/parca-dev/parca-agent-test:' + version,
+    image: 'ghcr.io/parca-dev/parca-agent-test:' + self.version,
     // This assumes there's a running parca in the cluster.
```

Comment:
Not required for this PR but if we want to use image artifacts here instead to fix #934 then we can do away with this bit as well as we won't need `ghcr.io/parca-dev/parca-agent-test` any longer.